rest/route/patch_test.go

```diff
@@ -829,9 +830,17 @@ buildvariants:
 		Id:                   mgobson.NewObjectId(),
 		Project:              projectRef.Id,
 		Githash:              "3c7bfeb82d492dc453e7431be664539c35b5db4b",
-		PatchedParserProject: config,
+		ProjectStorageMethod: evergreen.ProjectStorageMethodDB,
+		PatchedProjectConfig: config,
 	}
 	require.NoError(t, unfinalized.Insert())
+
+	pp := &serviceModel.ParserProject{}
+	err := util.UnmarshalYAMLWithFallback([]byte(config), &pp)
+	require.NoError(t, err)
```

Comment:
nit: You can just combine L839-840.

Same with below.

Reply:
done